onnxruntime/core/providers/rocm/rocm_pch.h.orig

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
was this file unintentionally checked in?

Reply:
Addressed